services/QuillLMS/client/app/bundles/Staff/components/comprehension/activities.tsx

```diff
@@ -47,6 +52,8 @@ const Activities = ({ location, match }) => {
 
   return(<React.Fragment>
     <Navigation location={location} match={match} />
+    {hasAppSetting && <div>App setting is enabled.</div>}
+    {errors && renderErrorsContainer(false, errors)}
```

Comment:
This is not necessary for the demo, and there will be times when we want an app setting to fail silently (as long as the client API returns a value of false), but Eric suggested explicit error message displaying for this component.